js/unit_tests.ts

```diff
@@ -13,6 +13,34 @@ test(async function tests_test() {
   assert(true);
 });
 
+// TODO Add tests for modified, accessed, and created fields once there is a way
+// to create temp files.
+test(async function tests_statSync() {
+  const packageInfo = deno.statSync("package.json");
+  assert(packageInfo.isFile());
+  assert(!packageInfo.isSymlink());
+
+  const testingInfo = deno.statSync("testing");
+  assert(!testingInfo.isDirectory());
+  assert(testingInfo.isSymlink());
+
+  const srcInfo = deno.statSync("src");
+  assert(srcInfo.isDirectory());
+  assert(!srcInfo.isSymlink());
+
+  let caughtError = false;
+  let badInfo;
+
+  try {
+    badInfo = deno.statSync("<bad_file_name>");
+  } catch (e) {
+    caughtError = true;
+  }
+
+  assert(caughtError);
+  assertEqual(badInfo, undefined);
```

Comment:
Maybe break this into a separate test? Also rename the tests: 
```
tests_statSync() -> statSyncSuccess
```
and the new one, call it `statSyncNotFound`

Also do
```
  // TODO assert(err instanceof deno.NotFound).
  assert(err);
  assertEqual(err.name, "deno.NotFound");
```